pkg/apis/policy/v1alpha1/propagation_types.go

```diff
@@ -173,6 +173,13 @@ type PropagationSpec struct {
 	// +kubebuilder:validation:Enum=Lazy
 	// +optional
 	ActivationPreference ActivationPreference `json:"activationPreference,omitempty"`
+
+	// Suspend will instruct all work objects to pause propagation to member clusters.
+	// Defaults to false.
+	//
+	// +kubebuilder:default=false
+	// +optional
+	Suspend bool `json:"suspend,omitempty"`
```

Comment:
Do we need to consider setting pause fields for different clusters so that different pause policies can be set for different work of the same resource?

/cc @CharlesQQ @RainbowMango